### ISSUE TITLE
b-dropdown-item fails accessibility as role is not defined

### ISSUE THREAD

**Issue by adholland** [bootstrap-vue-next/bootstrap-vue-next#1412]
package: 0.11.0

b-dropdown-item fails accessibility as role is not defined. To overcome the issue I am currently using the following workaround: -

```
mounted() {
    const items = this.$el.querySelectorAll('.tool-bar-component#tool-bar-component-id ul>li>a.dropdown-item');
    items.forEach(element => element.role = 'menuitem');
},
```

**Comment by VividLemon** [comment]
It's like one line, how difficult is it to make a PR
